ArduPlane/geofence.cpp

```diff
@@ -183,9 +183,17 @@ void Plane::geofence_update_pwm_enabled_state()
     }
 
     if (geofence_state->is_pwm_enabled != is_pwm_enabled) {
+        if (is_pwm_enabled) {
+        gcs().send_text(MAV_SEVERITY_INFO, "GeoFence Enabled");
```

Comment:
Indenting issues.

Also, this would properly go inside the `geofence_set_enabled` function - or possibly deeper.